homeassistant/components/switchbot/config_flow.py

```diff
@@ -134,6 +134,9 @@ async def async_step_user(
     async def async_step_import(self, import_config: dict[str, Any]) -> FlowResult:
         """Handle config import from yaml."""
         _LOGGER.debug("import config: %s", import_config)
+        _LOGGER.warning(
+            "Yaml config for Switchbot has been deprecated and will be removed in 2022.4. Please remove it from your config"
```

Comment:
Normally we log in the place that starts the config flow, eg the `async_setup_platform` switch platform in this case.